assets/sass/v2/_select-authenticator.scss

```diff
@@ -0,0 +1,19 @@
+
+.siw-main-view {
+  .okta-form-subtitle {
+    text-align: center;
```

Comment:
should be left aligned.

Reply:
These are the subtitles that I see are center aligned in the mocks 
![Screen Shot 2020-06-09 at 4 32 38 PM](https://user-images.githubusercontent.com/23267876/84210931-e565cd00-aa6e-11ea-90f5-c983dda96b74.png)
![Screen Shot 2020-06-09 at 4 32 50 PM](https://user-images.githubusercontent.com/23267876/84210934-e991ea80-aa6e-11ea-8df2-05f3e9463803.png)